web-animations-2/Overview.bs

```diff
@@ -318,7 +322,6 @@ follows:
 
```

Comment:
I cannot see where your links are pointing to.

Reply:
Only one of the two links is working for me.  

Referring to the handling of start time in the set timeline procedure, we have a few options:
* When switching timelines, we always set auto align start time.  This means that setting current time than then switching timelines won't preserve current time.  Though as the units of time are changing, all we would be able to preserve is progress.
* When switching timelines, we preserve progress if auto-align is false, or the new timeline is time-based, otherwise hands off.  This means that we should either make auto-alignment 3 state (true|false|unset) and only set to true if in the unset state, or always set auto-align to true when calling play regardless of the timeline type (it simply won't get used while time-based). Then when we switch to a scroll-based animation, the flag will already be set if current time or start time have not been called.

Reply:
Cleaned up the procedure for setting the timeline based on offline discussion.